src/ui/localization/localized.svelte

```diff
@@ -0,0 +1,35 @@
+<script lang="ts">
+  import { Translator } from "./translator";
+  import { getContext } from "svelte";
+  import { translatorContextKey } from "./constants";
+
+
+  export interface LocalizedProps {
```

Comment:
this should be defined elsewhere to be exported or we need to change this to a module

Reply:
Fixed it, just me being clumsy with Svelte.

I just wanted it to be the props of this component, no need to export them.